components/StatusModal.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div ref="statusDialog" class="text-center">
+        <v-dialog v-model="dialog" width="500">
+            <v-card>
+                <v-card-title class="text-h5 grey lighten-2">
+                    {{title}}
+                </v-card-title>
+                <br />
+                <v-card-text>
+                    {{text}}
+                </v-card-text>
+                <v-divider></v-divider>
+                <v-card-actions>
+                    <v-spacer></v-spacer>
+                    <v-btn color="primary" text @click="close">Ok!</v-btn>
```

Comment:
There's actually some unofficial naming conventions for event handling!

### For events
the `action` itself is fine. `click`, `tapped`, `tap`, `scrollup`, etc.  (Personally I prefer past tense....`clicked` vs. `click`.

### For event handlers
You prefix the event with `handle` like `handleClicked` or `handleClosed`.

### For event hooks/functions props called when an event happens
You prefix the event with `on` like `onClicked` or `onClosed`.

#### How are event hooks/ function props different than a handler? 
Honestly, these are often the same thing, but sometimes they are distinct. 
They are typically delegate methods that might be run when the event happens but don't guarantee the event has been "handled".  Attaching logging is a good example

cc @theyokohamalife